src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -25,7 +25,9 @@ import no.skatteetaten.aurora.boober.service.CantusService
 @Service
 class AzureFeature(
     cantusService: CantusService,
-    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String
+    @Value("\${clinger.sidecar.default.version:0}") val sidecarVersion: String,
+    @Value("\${clinger.sidecar.default.ldapurl:}") val defaultLdapUrl: String,
```

Comment:
Er det greit at ldapurl her blir tom streng hvis den ikke er spesifisert? Eller bør appen feile under oppstart?

Reply:
_boober_ skal ikke feile fordi den ikke er satt. Men applikasjonen skal muligvis (det er mulig å override den). 

Jeg ser nå at vi ikke faktisk sjekker at verdien er satt korrekt (men vi tar den først i bruk hvis den er satt og gyldig). 
Det bør nok sjekkes at applikasjoner som sier at de vil bruke gruppe-oppslag har satt ldapUrl til noe fornuftig (eller at defaulten er det ellers).

Reply:
Ha laget validering der det klages fra boober dersom delvis konfigurasjon er satt opp for iv-groups-støtte.